src/index.html

```diff
@@ -9,10 +9,58 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="../src/styles/style.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
+      <div class="stars--0">
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+
+      <div class="stars--1">
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+
+      <div class="stars--2">
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+
+      <div class="stars--3">
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+
+      <div class="stars--4">
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
+
+      <div class="stars--5">
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
```

Comment:
The class names like `stars--0`, `stars--1`, etc., do not follow the BEM methodology correctly. Consider using BEM naming conventions, such as `stars__item--0`, to improve clarity and maintainability.